webapp/channels/package.json

```diff
@@ -52,7 +52,7 @@
     "lodash": "4.17.21",
     "luxon": "3.3.0",
     "mark.js": "8.11.1",
-    "marked": "github:mattermost/marked#2ef7f28cc7718e3f551c4ce9ea75fdd7580c2008",
+    "marked": "github:willypuzzle/marked#8dba39e0cee718d19dbb4cf5417d5155f6f7d48b",
```

Comment:
We maintain our own fork of marked so feel free to update to https://github.com/mattermost/marked

Reply:
I think you should approve and merge the related PR, showed in the description